api/libproject/devstart.rb

```diff
@@ -1914,7 +1910,7 @@ def describe_runtime(cmd_name, *args)
     common = Common.new
     common.run_inline %W{
        ./gradlew manageLeonardoRuntimes
-      -PappArgs=['describe','#{api_url}','#{gcc.project}','#{ctx.service_account}','#{op.opts.runtime_id}']}
```

Comment:
not just unused, but wrong: this was attempting to pass the `environment` as the workspace google proejct